src/lib/vocab/policy/policyHelpers.server.ts

```diff
@@ -47,3 +48,17 @@ export const checkCommunityAccess = async (
 		return {ok: false, status: 403, message: 'actor does not have permission'};
 	}
 };
+
+export const checkDisabledCommunityCreation = async (repos: Repos): Promise<boolean> => {
+	const result = unwrap(await repos.community.findById(ADMIN_COMMUNITY_ID));
+
+	if (!result) {
+		throw Error('unable to locate admin community');
+	}
+
+	if (result.settings.instance) {
+		return !!result.settings.instance.disableNewCommunities;
+	} else {
+		return false;
+	}
```

Comment:
stylistic choice, could be shortened:

```suggestion
	return !!result.settings.instance?.disableNewCommunities;
```

Reply:
I pretty much always choose the more verbose form because I'm not used to thinking in truthy/falsey terms so it makes it easier for me to reason about it when I write it. Not opposed to shorter form though lol